eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -277,6 +289,36 @@ jobs:
 
       - ${{ parameters.PreTestSteps }}
 
+      # JRSSTART
```

Comment:
I assume you planned to remove this JRSSTART comment.

Reply:
Removed with 5th commit.